lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -91,8 +95,8 @@
       <li class="action action-share">
         <a class="action-linkedin-profile" target="_blank" href="${cert_status['linked_in_url']}"
          title="${_('Add Certificate to LinkedIn Profile')}"
-         data-course-id="${unicode(course_overview.id)}"
-         data-certificate-mode="${cert_status['mode']}"
+         data-course-id="${course_overview.id | n, unicode}"
```

Comment:
I don't have a strong opinion on this. I just saw on the "safe_templates" doc we can use mako json filter. But I do agree we don't need to update this